pysteps/verification/salscores.py

```diff
@@ -302,10 +302,10 @@ def _sal_l2_param(prediction, observation, thr_factor, thr_quantile, tstorm_kwar
     )
     obs_r = (
         _sal_weighted_distance(observation, thr_factor, thr_quantile, tstorm_kwargs)
-    ) * (np.nanmean(observation))
+    )
     forc_r = (
         _sal_weighted_distance(prediction, thr_factor, thr_quantile, tstorm_kwargs)
-    ) * (np.nanmean(prediction))
+    ) 
```

Comment:
same comment as above, note that the extra blank character at line 308 is causing the code formatting check to fail -> remember to run `black .` before committing your changes (see section "Using Black auto-formatter" in our [Contributor guidelines](https://pysteps.readthedocs.io/en/stable/developer_guide/contributors_guidelines.html#code-style)).